src/components/CustomTableRow/CustomTableRow.tsx

```diff
@@ -0,0 +1,36 @@
+import { Checkbox, TableCell, TableRow } from '@mui/material';
+import { CustomTableRowProps } from './CustomTableRowProps.types';
+import { MouseEvent } from 'react';
+
+export function CustomTableRow<T extends Record<PropertyKey, unknown>>(props: CustomTableRowProps<T>) {
+  const { row, columnConfig, rowIdx, isSelected, handleClick } = props;
+  const labelId = `enhanced-table-checkbox-${rowIdx}`;
+
+  return (
+    <TableRow
+      hover
+      onClick={(event: MouseEvent<unknown>) => {
+        return handleClick(event, row);
+      }}
+      role='checkbox'
+      aria-checked={isSelected}
+      tabIndex={-1}
+      key={row.id as string}
```

Comment:
same story

Reply:
fixed in [44e72af](https://github.com/dorm-social-room-manager/dsrm-frontend/pull/38/commits/44e72afe781fbe2467eebdb5c74be32d6133e4b1)